src/tests/ftest/cart/iv/cart_iv_two_node.py

```diff
@@ -22,6 +22,11 @@
   portions thereof marked with this legend must also reproduce the markings.
 '''
 
+# pylint: disable=import-error
+# pylint: disable=broad-except
```

Comment:
We probably want to catch `OSError`, instead of `OSError`'s parent: `Exception`. In case a sub-class of `OSError` is raised, is there a way to polymorphically catch an exception in python? Here's the `OSError` class hierarchy:

```
+-- OSError
|    +-- BlockingIOError
|    +-- ChildProcessError
|    +-- ConnectionError
|    |    +-- BrokenPipeError
|    |    +-- ConnectionAbortedError
|    |    +-- ConnectionRefusedError
|    |    +-- ConnectionResetError
|    +-- FileExistsError
|    +-- FileNotFoundError
|    +-- InterruptedError
|    +-- IsADirectoryError
|    +-- NotADirectoryError
|    +-- PermissionError
|    +-- ProcessLookupError
|    +-- TimeoutError
```

Disabling `broad-except` more block-specifically for now: d6236cc4be6f468a7a22e9701f2ce9d3ed4ffda1.